ProcessLib/HeatTransportBHE/BHE/CreateBHE1PType.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "BaseLib/ConfigTree.h"
+#include "CreateBHEUType.h"
+
```

Comment:
The map, memory etc includes are probably needed here...

Reply:
Added in the header file.